### DISCUSSION TITLE
Hotspots on target and scaffold protein for binder desing?

### DISCUSSION THREAD

**Discussion by Cookiemaster33** [RosettaCommons/RFdiffusion#157] [General]
Hi there.
I really love the binder design function, especially coupling it with fold conditioning. I had some good results designing a protein that can bind to the hotspot regions on my target protein. 

What I would love to do however, is to design binder proteins that are similar to an input structure, and at the same time bind the hotspot of my target just with a certain domain. Unfortunately, rfdiffusion spits out a lot of binders that bind the target with other domains. Thus, I need to do a lot of quality control and have to sift through structures to find "the needle in the haystack".  (Does this make sense somehow?)

I have tried to play around with potentials, but it unfortunately went nowhere. 
I think It would be fantastic if I could define hotspots, not only on my target protein, but also on the input pdbs that are used for conditioning. I know that rfdiffusion only uses secondary structure constrains to do the conditioning, so I understand that this might be a bit difficult to implement. But if there would be a solution for that in the future, I think that would be helpful to other people to (?). On the other hand, I maybe use the program wrong :D

**Comment by kaiwanxiao1996** [comment]
"Hello, I hope you're doing well. I was wondering if you might know the specific process for using secondary structure constraints to perform conditioning. I found in the repository at https://github.com/RosettaCommons/RFdiffusion that it provides constraints for the target’s secondary structure during binder design. However, in Baker's latest paper, they mention using conditioning to constrain the binder’s secondary structure when binding to the target. Specifically, they stated: 'In order to generate desired binder-target β-strand pairing interactions using RFdiffusion, fold-conditioning tensors describing single binder β-strands interacting with target β-strands in a matrix format were supplied to RFdiffusion at inference.' Could you kindly explain how this is implemented? I would greatly appreciate any insights you could offer. Thank you so much!"
